src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -253,19 +253,41 @@ void LogConfiguration::configure_output(size_t idx, const LogSelectionList& sele
     on_level[level]++;
   }
 
+  // MT-Safety
+  // ConfigurationLock can only guarantee that only one thread is executing reconfiguration. This function still must
+  // be MT-safe because logsites in other threads may be executing in parallel.
+  //
+  // Reconfiguration: Unified logging allows users to dynamically change tags and decorators of a log output.
+  //
+  // A synchronization 'wait_until_no_readers()' is imposed inside of 'ts->set_output_level(output, level)' above
+  // if setting has changed. It guarantees that all logs either synchronous writing or enqueuing to the async buffer
+  // see the new tags and decorators. It's worth noting that the synchronization happens even level doesn't change.
+  //
+  // LogDecorator is a set of decorators represented in a uint. sizeof(uinit) is not greater than a machine word,
+  // so store of it is atomic on the mainstream processors. I.e. readers see either its older value or new value.
```

Comment:
We should use Atomic::store and Atomic::load regardless.

Reply:
I just comment out why it's MT-safe here. if it's necessary, we can use a new JBS issue to change it to Atomic::store.